app/javascript/src/utilities/ReCaptchaV3.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+
+import { useScript } from 'utilities/useScript'
+
+import type { ReactElement } from 'react'
+
+interface Props {
+  enabled?: boolean;
+  siteKey: string;
+  action: string;
+}
+
+const ReCaptchaV3 = ({ enabled = false, siteKey, action }: Props): ReactElement => {
+  if (!enabled) return (<div>{null}</div>)
```

Comment:
```suggestion
  if (!enabled) {
    return <div>{null}</div>
  }
```

Also, why `<div>{null}</div>` and not  `null`?

Reply:
Because the linter fails with:

```
TS2322: Type 'null' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.
```